python-sdk/src/astro/custom_backend/serializer.py

```diff
@@ -87,12 +87,5 @@ def _attempt_to_deser_unknown_object(obj: str):
         log.debug("Attempting to deserialize object %s into a json object", obj)
         return json.loads(obj)
     except JSONDecodeError:
-        try:
-            log.debug(
-                "Json debugging failed for object %s, attempting to pickle deserialize",
-                obj,
-            )
-            return pickle.loads(bytes.fromhex(obj))
-        except UnpicklingError:
-            log.debug("unpickling failed for object %s, returning raw object", obj)
-            return obj
+        log.debug("Json deserializing failed for object %s, returning raw object", obj)
+        return obj
```

Comment:
Why are we returning the `obj` when `JSONDecodeError`?

Reply:
@sunank200, if we don't know how to debug an object, we have to assume that the user has an object that is not a json object and let them decide how to handle it.